Testscripts/Linux/KVP-BasicChecks.sh

```diff
@@ -41,7 +41,7 @@ fi
 pid=$(pgrep "hypervkvpd|hv_kvp_daemon")
 if [ $? -ne 0 ]; then
     LogErr "KVP Daemon is not running by default"
```

Comment:
please change this to LogMsg.
Also do the same message with UpdateSummary, so that we have it handy.